### ISSUE TITLE
Output is really hard to read

### ISSUE THREAD

**Issue by sbrl** [tldr-pages/tldr-node-client#39]
I'm finding the output rather difficult to read on Ubuntu. Here's a screenshot:

![tldr output](http://i.imgur.com/0dspAF9.png)

The repo doesn't specify which colour names are valid, and so I don't know what to change it to in my `.tldrrc` file.

**Comment by igorshubovych** [comment]
If you are using tldr-node-client, create empty .tldrrc in your HOME folder, put there a JSON from tldr-node-client README.md

**Comment by sbrl** [comment]
Done, produces the same output.

**Comment by igorshubovych** [comment]
This is my .tldrrc file.
~~~JSON
{
  "colors": {
    "text": "green",
    "command-background": "white",
    "command-foreground": "red",
    "command-token": "white"
  }
}
~~~

You can change some colors and check if they affects the rendering.

The valid colors are in npm [colors](https://www.npmjs.com/package/colors) package.

**Comment by sbrl** [comment]
Thanks! That really helped. Here's my new config:

```json
{
  "colors": {
    "text": "green",
    "command-background": "white",
    "command-foreground": "red",
    "command-token": "blue"
  }
}
```

White on white didn't show up for me, so I changed it to blue.

**Comment by igorshubovych** [comment]
Can you plz share your screenshot?

Maybe we can put your configuration in README.md.

**Comment by sbrl** [comment]
![1](http://i.imgur.com/csZFfbS.png)

**Comment by rprieto** [comment]
Should we change the default config to be text-only? This issue seems to come up more and more, even on the main `tldr` "pages" repo. I think it will be hard to have defaults that suit everyone.

**Comment by sbrl** [comment]
Hmmm. Having an option for text only might be a good idea. Personally, I'd like to see an option for a transparent background, too.

**Comment by igorshubovych** [comment]
Give me a few days. I am working on themes.

**Comment by sbrl** [comment]
Sure thing!

**Comment by rprieto** [comment]
How will themes work @igorshubovych? I'm curious if we'll find something that works for everyone regardless of people's terminal background color.

Maybe we should open a new issue to discuss color-related things & rendering in general, since there were so many issues on the topic - then we can link any related ticket to that one.
- https://github.com/tldr-pages/tldr-node-client/issues/18
- https://github.com/tldr-pages/tldr-node-client/issues/39
- https://github.com/tldr-pages/tldr/issues/388
- https://github.com/tldr-pages/tldr/issues/323
- https://github.com/tldr-pages/tldr/issues/270
- https://github.com/tldr-pages/tldr/issues/256
- https://github.com/tldr-pages/tldr/issues/209
- https://github.com/tldr-pages/tldr/issues/96
- https://github.com/tldr-pages/tldr/issues/59

**Comment by glortho** [comment]
Looking forward to themes! In the meantime, here's a very barebones version: https://github.com/glortho/tldr-node-client 
<img width="546" alt="screen shot 2015-12-29 at 09 51 20" src="https://cloud.githubusercontent.com/assets/526284/12036198/c8f0974e-ae11-11e5-93b8-0b83c7a26c1b.png">

**Comment by glortho** [comment]
Looking forward to theme support! In the meantime, here's a minimalist non-configurable transparent version: https://github.com/glortho/tldr-node-client

<img width="412" alt="screen shot 2015-12-29 at 09 55 32" src="https://cloud.githubusercontent.com/assets/526284/12036275/6c58eb70-ae12-11e5-8902-325d14407e4b.png">

**Comment by sbrl** [comment]
@glortho Installed.

**Comment by igorshubovych** [comment]
@glortho,

I really like your output.
Maybe you should squash your commits and submit a pull request?
I personally don't mind if default looks like this. Let's see what @rprieto thinks.

**Comment by rprieto** [comment]
I really like it as well! You're right it seems more legible without a background.

How did you achieve transparency? I can see your config says `background: black`, does it mean it only works because you have a black terminal window? (but I'm confused because it looks grey :smile:). If that's the case we need to implement actual transparency, e.g. `background: null`.

I'd be happy for this style to be the default "colored" theme too. But I don't think we should merge the PR just yet though given the work on themes... otherwise we're open to many people submitting their theme as the new better non-configurable default. Let's make sure we come back when themes are implemented!

**Comment by kkarpieszuk** [comment]
@igorshubovych any solution for tldr installed with pip/python? Will .tldrrc file work too?

**Comment by sbrl** [comment]
@kkarpieszuk Perhaps you should file a bug on tldr-pages/tldr-python-client.

**Comment by sbrl** [comment]
@rprieto I think that the text background is transparent by default, if you don't apply a background.

**Comment by igorshubovych** [comment]
@kkarpieszuk there are 2 Python clients for TLDR. Check them out
https://github.com/tldr-pages/tldr#clients

I used one of them and it uses YAML config file instead of JSON, so be careful.

Both authors accept pull requests, so if you know Python, go and do it.

**Comment by agnivade** [comment]
Fixed by #117
